.tool-versions

```diff
@@ -0,0 +1,5 @@
+yarn 1.22.19
+nodejs 16.20.1
```

Comment:
The latest nodejs LTS is `18.12.0` - do we use an older one for compatibility reasons?

Reply:
actually - the `next` package throws an error: 
```
next@14.1.0: The engine "node" is incompatible with this module. Expected version ">=18.17.0". Got "16.20.1"
```
so we'll need to update our version of node

Reply:
I was trying to keep consistent with registration where possible. I'll update the node version though.